paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -349,6 +344,27 @@ public class AccessibilityRenderExtension : RenderExtension {
     )
   }
 
+  private fun SemanticsNode.hasZeroAlpha(): Boolean {
+    // try to reflectively query internal SemanticsNode.isTransparent() by iterating on kotlin field mangled names:
+    val candidateGetterNames = arrayOf(
+      "isTransparent",
+      "isTransparent\$ui_release",
+      "getIsTransparent",
+      "getIsTransparent\$ui_release"
+    )
+    for (name in candidateGetterNames) {
```

Comment:
Feels like this is near inner loop; executed on all nodes. I should be possible to cache this reflection filter query (companion object by lazy?) and only doing the invoke in hasZeroAlpha.